ipa-multipoint/src/test/java/org/hyperledger/besu/nativelib/ipa_multipoint/proof/ExecutionWitnessData.java

```diff
@@ -0,0 +1,113 @@
+package org.hyperledger.besu.nativelib.ipa_multipoint.proof;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+
+public class ExecutionWitnessData {
+
+    @JsonProperty("header")
+    public Header header;
+    @JsonProperty("transactions")
+    public List<String> transactions;
+    @JsonProperty("withdrawals")
+    public List<String> withdrawals;
+    @JsonProperty("executionWitness")
+    public ExecutionWitness executionWitness;
+
+
+    static class Header {
```

Comment:
I think these nested classes can be record types without the additional serialization annotations, using @JsonSerialize annotation on ExecutionWitnessData.  NBD, just less verbose

Reply:
for the moment the project is still with java 11 from what I see so I cannot use record